src/Scripting/CSharp/Hosting/CommandLine/Csi.cs

```diff
@@ -18,15 +18,21 @@ internal CSharpInteractiveCompiler(string responseFile, string baseDirectory, st
         {
         }
 
+        internal override Type Type => typeof(CSharpInteractiveCompiler);
+
+        internal override string GetAssemblyFileVersion()
+        {
+            return Type.GetTypeInfo().Assembly.GetCustomAttribute<AssemblyFileVersionAttribute>().Version;
```

Comment:
"I don't know" is a perfectly valid comment. It cleanly instills fear in the next person. :smile: